src/components/BrowseSkill/BrowseSkill.js

```diff
@@ -121,22 +121,24 @@ export default class BrowseSkill extends React.Component {
           this.setState({ groups: data });
           data.sort();
           groups.push(
-            <MenuItem
-              value="All"
-              key="All"
-              primaryText="All"
-              onClick={event => this.handleGroupChange(event, 'All')}
-            />,
+            <Link to="/category/All" key="All">
+              <MenuItem
+                value="All"
+                primaryText="All"
+                onClick={event => this.handleGroupChange(event, 'All')}
```

Comment:
onClick event is no longer needed now that we are transitioning the page, please remove. :+1: